config/application.rb

```diff
@@ -27,6 +27,6 @@ class Application < Rails::Application
     # config.eager_load_paths << Rails.root.join("extras")
 
     config.session_store :cookie_store, key: "_rails_world_session", expire_after: 1.month
-    config.time_zone = "Eastern Time (US & Canada)"
+    config.time_zone = ENV["TIME_ZONE"] || "UTC"
```

Comment:
We would need to make sure that we update the .env files in our current staging and prod environments so it doesnt default to UTC